Secret-agency-service/src/main/java/cz/fi/muni/pa165/secretagency/service/facade/ReportFacadeImpl.java

```diff
@@ -63,7 +63,9 @@ public void createReport(ReportCreateDTO reportDTO) {
 //        report.setDate(reportDTO.getDate());
 //        report.setText(reportDTO.getText());
 //
-//        reportService.save(report);
+//        Report savedReport = reportService.save(report);
+//        return savedReport.getId();
+        return 0L;
```

Comment:
Asi bych s timto pockal, az bude mission service, a pak na to navazal.
Menit temporary kod za jiny temporary kod nema podle me moc smysl